<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_Button_PluginItem</class>
 <widget class="QWidget" name="Widget_Button_PluginItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget_Button_PluginItem{
	background-color: rgb(245, 245, 245);
}

#label_pluginName{
	color: rgb(0,0,0);
}

#label_filePath{
	color: rgb(79, 79, 79);
}

/*附加插件标记标签*/
#label_sign,#label_version{
	color: rgb(28, 28, 28);
	background-color: rgb(255, 255, 255);
	border-radius:8px;
	padding:4px;
	margin-left:8px;
	margin-top:5px;
	margin-bottom:5px;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_img">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../icons.qrc">:/WidgetIcon/icon/WidgetIcon/plugin/pluginNormal.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_pluginName">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>默认按钮标题</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_sign">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Normal Sign</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_version">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>0.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_filePath">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;home/dirname/plugin.plg&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="But_enable">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>2</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>禁用</string>
      </property>
     </widget>
     <widget class="QPushButton" name="But_unload">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>30</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>卸载</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
